src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -21,7 +21,7 @@ commands:
 artifactsDir: tests/_e2e_artifacts_v2
 timeout: 360
 reportFormat: xml
-parallel: 4
+parallel: 1
```

Comment:
Why lowering parallelism?

Reply:
We are running on only 1 namespace, if we run in parallel the tests start occurring on the same namespace and the results get mixed up.